sections/reviews.md

```diff
@@ -0,0 +1,74 @@
+# Code reviews
+
+At Open Knowledge International (OKI) all code must be reviewed by a colleague before merging into the main branch of the repository (usually `master`). This short document examples how to do code reviews, and why we do it.
+
+## How
+
+The process for code reviews is straight forward as long as you are already following our coding standards for working with version control, and in particular our git flow-influenced process of branch management. The bottom line is that, before any code hits `master`, it must be reviewed by a colleague.
+
+### Submitting code for review
+
+Submitting code for review should follow these steps:
+
+1. Work out of your own fork of the main repository (usually `origin`).
+2. Always, always work from a feature or fix branch that is checked out of `master`.
+3. Each feature or fix branch should be focused on a discrete unit of work.
+4. When your unit of work is complete, submit a pull request against the `master` branch on `origin`.
```

Comment:
Make sure the PR has a description explaining the nature of the change, in technical terms.

i.e. not 'fixing the multi-view feature' but 'adding exception handling to the DB migrations'

(no, the name of the branch or title of the commit are usually not enough...)